SWIPs/swip-honey_to_money.md

```diff
@@ -7,51 +7,66 @@ Type: Standards track
+<!--The rationale fleshes out the specification by describing what motivated the design and why particular design decisions were made. It should describe alternate designs that were considered and related work, e.g. how the feature is supported in other languages. The rationale may also provide evidence of consensus within the community, and should discuss important objections or concerns raised during discussion.-->
+### General rationale for choosing oracles
+There exist a need for the price of Swarm to change over time. However, due to the nature of Swarm, there is a need for this price to be decided network-wide, instead of peer-to-peer. Specifically:
+- The requirement on connectivity in the local bin and far-away bins means that nodes *must agree* with all nodes in their local bin and at least one node in a far-away bin on the price.
+- Zero marginal costs of serving data with perfect competition means that when nodes would engage in peer-to-peer price discovery, the price will tend to zero and a sub-optimal equilibrium of near-zero prices will be reached.
+- The high-quantity of transactions implies that nodes cannot engage in price-discovery for every byte of data transacted at the point of transaction, meaning they have to agree *beforehand* on how they will come to a price *at a later time*. 
+- Nodes are most of the time just conduits for data and payments, meaning that most the time data requests and thus also payments will be because you want to send data to a peer who requested this data from you. 
+### Rationale for the described oracle specification
+* Nodes might apply different honey prices for the same payment, due to the time difference between querying the oracle for *sending* a payment and querying the oracle for *receiving* the same payment. The imbalances caused by this don't harm if the sum of all inbalances is not greater than the disconnect threshold. Experimentation should confirm that this indeed does not happen. Theoretically, if the overall price movement does not show a clear trend (either up or down) over the period where nodes do business with each other or the two nodes are neither a net receiver nor a net sender to each other, accounting imbalances should offset each other. Furthermore, less frequent as well as small price updates should minimize the imbalances. If a clear trend exists and one node is a net user and the other a net provider, the disconnect threshold will eventually be reached. Research must be performed on how the strength of a trend influences the duration until a disconnect is reached for a given stream of traffic. The assumption is that this will be sufficiently long not to cause any problems for a typical Swarm user. If there appears to be a problem with such imbalances, we see a possible solution:
```

Comment:
(thinking out loud: is `Swarm user` as correct as `Swarm node`?)

Reply:
Hmm... I would say not, as a Swarm user might be a light client or some other sort of auxiliary user. A Swarm node engages in SWAP, which a Swarm user might not necessarily do. 
Is a change required?

Reply:
yeah, let's go ahead and change it to `Swarm node` then.

Reply:
(paragraph has been updated but the correction hasn't been made. lmk if you don't agree)